packages/sedlex/sedlex.2.1/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+synopsis: "An OCaml lexer generator for Unicode"
+description: "
+sedlex is a lexer generator for OCaml. It is similar to ocamllex, but supports
+Unicode. Unlike ocamllex, sedlex allows lexer specifications within regular
+OCaml source files. Lexing specific constructs are provided via a ppx syntax
+extension.
+"
+license: "MIT"
+doc: "https://ocaml-community.github.io/sedlex/index.html"
+maintainer: "Alain Frisch <alain.frisch@lexifi.com>"
+authors: [
+  "Alain Frisch <alain.frisch@lexifi.com>"
+  "https://github.com/ocaml-community/sedlex/graphs/contributors"
+]
+homepage: "https://github.com/ocaml-community/sedlex"
+dev-repo: "git+https://github.com/ocaml-community/sedlex.git"
+bug-reports: "https://github.com/ocaml-community/sedlex/issues"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+  ["dune" "build" "@doc" "-p" name "-j" jobs] {with-doc}
+]
+depends: [
+  "ocaml" {build & >= "4.02.3"}
```

Comment:
```suggestion
  "ocaml" {>= "4.02.3"}
```
I would also avoid having ocaml be a build dependency but I'm not too sure on this one.

Reply:
Well, de facto, depending on packages that have a non-build dependency on `ocaml` makes this package also having a non-build dependency on `ocaml`..